guides/common/assembly_api-requests-in-various-languages.adoc

```diff
@@ -2,6 +2,20 @@ include::modules/con_api-requests-in-various-languages.adoc[]
 
 include::modules/proc_calling-the-api-in-curl.adoc[leveloffset=+1]
 
+include::modules/proc_passing-json-data-to-the-api-request.adoc[leveloffset=+2]
+
+include::modules/proc_retrieving-a-list-of-resources.adoc[leveloffset=+2]
+
+include::modules/proc_creating-and-modifying-resources.adoc[leveloffset=+2]
+
 include::modules/proc_calling-the-api-in-ruby.adoc[leveloffset=+1]
 
+include::modules/proc_creating-objects-by-using-ruby.adoc[leveloffset=+2]
+
+include::modules/proc_using-apipie-bindings-with-ruby.adoc[leveloffset=+2]
+
 include::modules/proc_calling-the-api-in-python.adoc[leveloffset=+1]
+
+include::modules/proc_creating-objects-by-using-python.adoc[leveloffset=+2]
+
+include::modules/proc_requesting-information-from-the-api-by-using-python.adoc[leveloffset=+2]
```

Comment:
title is slightly different to line 7. Suggestions welcome

Reply:
How about `Retrieving resource information by using Python`?